packages/grafana-ui/src/components/TimePicker/TimeRangePicker/TimeRangeForm.tsx

```diff
@@ -73,6 +73,10 @@ export const TimeRangeForm: React.FC<Props> = (props) => {
       const raw: RawTimeRange = { from: from.value, to: to.value };
       const timeRange = rangeUtil.convertRawToRange(raw, timeZone);
 
+      if (!timeRange.from.isBefore(timeRange.to)) {
+        return;
+      }
```

Comment:
This is a great check, but this is not a great user experience. Maybe we should warn the user that they have entered an incorrect order and make the form invalid?

Reply:
Seems a great suggestion @hugohaggmark ! 
Where do you think we should put that warning?

Reply:
Sorry for the late response but I think we would need to refactor this form quite a bit so we can present an error message directly in the form inputs if the data ranges are reversed. Let me know if you need help doing this.